sdk/tables/azure-data-tables/vcpkg/vcpkg.json

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+{
```

Comment:
Now all the manifest files have the comment like "all changes here will be lost", please put it here as well.